liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/diff.test.groovy

```diff
@@ -46,6 +46,10 @@ Optional Args:
     Default: null
   referenceDriverPropertiesFile (String) The JDBC driver properties file for the reference database
     Default: null
+  referenceLiquibaseCatalogName (String) Reference catalog to use for Liquibase objects
```

Comment:
@filipelautert what is the difference between referenceDefaultCatalogName and referenceLiquibaseCatalogName (the same for schema)?